src/export/StructureDefinitionExporter.ts

```diff
@@ -122,11 +135,10 @@ export class StructureDefinitionExporter {
       let structDef = cloneDeep(this.structDefs.find(sd => sd.name === type));
```

Comment:
This is only looking at `name`.  It seems that it should also look at `id` and `url` to be consistent w/ other resolve behavior?